csharp/ql/src/experimental/Security Features/backdoor/ProcessNameToHashTaintFlow.qhelp

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>This query detects code flow from PorcessName property on the Process class into a hash function.</p>
+    <p>Such flow is often used in code backdoors to detect runnig processes and compare them to an obfuscated list of antivirus processes to aviod detection.</p>
```

Comment:
I see how this query can be useful for non-Solorigate backdoors, but it'd probably be helpful to mention Solorigate as an example to give users context.

Reply:
Correct. Actually we created this one under a different folder (backdoor instead of campaign) for that reason. I will add a note in the description.